inginious/common/tasks_problems.py

```diff
@@ -181,6 +181,14 @@ def input_is_consistent(self, task_input, default_allowed_extension, default_max
 
     @classmethod
     def parse_problem(self, problem_content):
+        # Checking problem edit inputs
+        if len(problem_content["offset"]) != 0:
+            try:
+                offset = int(problem_content["offset"])
+                if offset < 1:
+                    raise Exception("Line offset must be positive!")
+            except ValueError:
+                raise Exception("Line offset must be an integer!")
         return Problem.parse_problem(problem_content)
```

Comment:
Beware here, because you return the result of the parent class method only, so if you want to store `offset` as a integer in the task descriptor, this won't do the job.